packages/fxa-event-broker/README.md

```diff
@@ -44,24 +44,47 @@ should terminate user login sessions that were established prior to the event.
 
 ### Profile Change
 
-Sent when a user has changed their profile data in some manner. Updates to their
-profile may include a new primary email address, display name, or 2FA status. This
-event does not include what changed and the profile data a service has access to
-may not show any changes if the data changed was outside the OAuth scope the service
-was granted.
+Sent when a user's profile data has changed in some manner. Updates to their profile may include a
+new primary email address, display name, picture, locale, metrics enabled status, 2FA enabled status,
+account locked status, or account disabled status.
 
-Services should update any cached profile data they hold about the user.
+This event does not necessarily include everything that changed and the profile data a service has
+access to may not show any changes if the data changed was outside the OAuth scope the service was
+granted. If this event is encountered, and there are no data fields, then services should refresh
```

Comment:
@bbangert @clouserw This part is a little questionable... In FXA-9412, there was an ask to send more info through this event. However, rereading FXA-9412 there is an assumption that the data was already being sent in this event, which it turns out wasn’t entirely accurate… Not sure if that matters or not.